### PR TITLE
add sample code

### PR DESCRIPTION
**Pull request by hlts2**
# WHY
we want sample code for subee.

# WHAT
[x] Create a sample that can be easily executed using docker

@izumin5210

### CODE REVIEW

**Comment by hlts2** [comment]
When you have time, please make a review:bow::bow:

@izumin5210

File: _examples/your-first-cloudpubsub-app/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Your first Google Cloud Pub/Sub app
+
+## Files
+
+- [main.go](main.go) - example source code
+- [docker-compose.yml](docker-compose.yml) - tool for defining and running multi container Docker applications, contains Golang, Google Cloud Pub/Sub Emulator
+- [go.mod](go.mod) - Go modules dependencies
+- [go.sum](go.sum) - Go modules checksums
+
+you can find more information about go.mod and go.sumat [Go wiki](https://github.com/golang/go/wiki/Modules)
```
Comment (izumin5210) [inline _examples/your-first-cloudpubsub-app/README.md]:
```suggestion
You can find more information about `go.mod` and `go.sum` at [Go wiki](https://github.com/golang/go/wiki/Modules)
```

File: _examples/your-first-cloudpubsub-app/main.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+		subscriber,
+		// Set SingleMessageConsumer implementation.
+		// If engine is created with a constructor for single message consumer type, you have to add SingleMessageConsumer implementation.
+		subee.SingleMessageConsumerFunc(
+			func(ctx context.Context, msg subee.Message) error {
+				payload := event{}
+
+				json.Unmarshal(msg.Data(), &payload)
+
+				logger.Info("Received event",
```
Comment (izumin5210) [inline _examples/your-first-cloudpubsub-app/main.go]: can you use std `log.Logger` here?

File: _examples/your-first-cloudpubsub-app/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Requirements
+
+To run this example you will need Docker and docker-compose installed. See installation guide at https://docs.docker.com/compose/install/
+
+## Running
+
+```bash
+> docker-compose up
+[a lot of Cloud Pub/Sub logs...]
+golang_1     | {"level":"info","ts":1551161785.525428,"caller":"subee@v0.1.0/engine.go:46","msg":"Start Pub/Sub worker"}
```
Comment (izumin5210) [inline _examples/your-first-cloudpubsub-app/README.md]: please update this 🙏

File: _examples/your-first-cloudpubsub-app/README.md (hunk 1 above)
Comment (hlts2) [reply _examples/your-first-cloudpubsub-app/README.md]: oh.., thank you!

**Comment by bgpat** [comment]
In this pull request, you implemented in separated directory `_examples`, although I want to see examples in godoc.org. Also, I would rather place `README.md` in the root directory.

File: _examples/your-first-cloudpubsub-app/README.md (hunk 3 above)
Comment (hlts2) [reply _examples/your-first-cloudpubsub-app/README.md]: thank you!

**Comment by izumin5210** [comment]
> I want to see examples in godoc.org

I think so too. 
we should write "getting started" for root `README.md` and `godoc.org` later.

**Comment by hlts2** [comment]
Fixed feedback. Please make review.:bow:

440a6fcd3366696428b818b168c470479f91d42c...389717c02fd704d5ca133a7345b5ab90f104dcd9

**Comment by hlts2** [comment]
Fixed feedback. Please make review.:bow:

440a6fcd3366696428b818b168c470479f91d42c...389717c02fd704d5ca133a7345b5ab90f104dcd9

@izumin5210

**Comment by hlts2** [comment]
thank you feedback.:bow:
I will write "getting started" for root `README.md` and `godoc.org` later.

@bgpat

**Pull request merged**